src/Symfony/Component/Form/Extension/Validator/Util/ServerParams.php

```diff
@@ -12,10 +12,19 @@
 namespace Symfony\Component\Form\Extension\Validator\Util;
 
 use Symfony\Component\Form\Util\ServerParams as BaseServerParams;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @deprecated since Symfony 5.1. Use {@see BaseServerParams} instead.
  */
 class ServerParams extends BaseServerParams
 {
+    public function __construct(RequestStack $requestStack = null)
+    {
+        trigger_deprecation('symfony/form', '5.1', 'The "%s" class will be removed in 6.0. Use "%s" instead.', __CLASS__, BaseServerParams::class);
```

Comment:
I think we usually put that outside the constructor, before the class definition.

Reply:
the ` will be removed in 6.0` part should be removed also.

Reply:
Done, thanks